assets/src/design-system/components/pill/index.js

```diff
@@ -58,30 +52,31 @@ const StyledTab = styled.button(
     &:focus {
       outline: none;
     }
+    div {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      box-sizing: border-box;
+      box-sizing: content-box;
```

Comment:
remove whichever one's the redundant one?

Reply:
Also do we have to reset the box sizing for children every time `themeHelpers.focusableOutlineCSS` is used?

If so, maybe we add some behavior to reset the children so they don't inherit it?
```
const focusableOutlineCSS = (color) => css`
  ...
  * {
    box-sizing: border-box;
  }
`;
```

Or is this just a specific case where the child div needs the different box-sizing?

Reply:
i love this idea! i think it'll really cut down on the overrides internal to the outline.